### DISCUSSION TITLE
Waiting over 10 days for a ticket reply

### DISCUSSION THREAD

**Discussion by J0ddy** [community/community#52566] [GitHub Education]
### Select Topic Area

General

### Body

I have been waiting for ten days and counting for a support ticket reply. Is this normal?
![https://i.jod.gg/YUXA1/hIRaSozE78.png](https://i.jod.gg/YUXA1/hIRaSozE78/raw.png)

The ticket is regarding issues with academic reverification. And I'm only bringing this up here because I don't know where else to ask and my verification expires in a few days time.

**Comment by J0ddy** [comment]
@queenofcorgis

**Comment by samus-aran** [comment]
Hi @J0ddy 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐

You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by J0ddy** [reply]
Hi, I didn't mention that It was rejected, just that I'm not getting a response.
When filling in the forum, it asks for my location, which doesn't work on my PC or my phone.

**Reply by samus-aran** [reply]
Hi @J0ddy 

When you say it doesn't work is there an error on the screen? 

Here are a few discussions that help students enable their browser location. Check if they help 👍 

https://github.com/orgs/community/discussions/50578
https://github.com/orgs/community/discussions/42246
https://github.com/orgs/community/discussions/50261

**Reply by J0ddy** [reply]
Thanks, I'll take a look
